src/dex-ui-components/presets/SwapTokensForm/SwapTokensForm.tsx

```diff
@@ -127,54 +141,49 @@ for ${formValues.tokenToReceive.amount.toFixed(6)} ${formValues.tokenToReceive.s
 
   /** Update receive amount based on new liquidity */
   useEffect(() => {
+    const tokenToTradeId = formValues.tokenToTrade.tokenMeta.tokenId ?? "";
+    const tokenToReceiveId = formValues.tokenToReceive.tokenMeta.tokenId ?? "";
+    const areTokensPaired = !isEmpty(tokenToTradeId) && !isEmpty(tokenToReceiveId);
     const updatedTokenToTrade = {
       ...formValues.tokenToTrade,
-      poolLiquidity: getPoolLiquidityForTokenSymbol(formValues.tokenToTrade.symbol, props.poolLiquidity),
+      poolLiquidity: areTokensPaired ? getPoolLiquidityForTokenId(tokenToTradeId, props.poolLiquidity) : undefined,
```

Comment:
Token State poolLiquidity values needed to be cleared out when a pair was unselected (0 or only 1 token symbol selected). Otherwise, the switch token functionality would result in incorrect form values.